pkg/cmd/login/login.go

```diff
@@ -0,0 +1,124 @@
+package login
+
+import (
+	"context"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/MakeNowJust/heredoc"
+	msg "github.com/aziontech/azion-cli/messages/login"
+	api "github.com/aziontech/azion-cli/pkg/api/personal_token"
+	cmdPersToken "github.com/aziontech/azion-cli/pkg/cmd/create/personal_token"
+	"github.com/aziontech/azion-cli/pkg/cmdutil"
+	"github.com/aziontech/azion-cli/pkg/logger"
+	"github.com/aziontech/azion-cli/pkg/token"
+	"github.com/aziontech/azion-cli/utils"
+	"github.com/pelletier/go-toml/v2"
+	"github.com/spf13/cobra"
+	"go.uber.org/zap"
+)
+
+func NewCmd(f *cmdutil.Factory) *cobra.Command {
+	var username, password string
+
+	cmd := &cobra.Command{
+		Use:   msg.Usage,
+		Short: msg.ShortDescription,
+		Long:  msg.LongDescription,
+		Example: heredoc.Doc(`
+		$ azion login --help
+		$ azion login --username fulanodasilva@gmail.com --password "senhasecreta"
+        `),
+		RunE: func(cmd *cobra.Command, args []string) error {
+
+			if !cmd.Flags().Changed("username") {
+				answers, err := utils.AskInput(msg.AskUsername)
+
+				if err != nil {
+					logger.Debug("Error while parsing answer", zap.Error(err))
+					return utils.ErrorParseResponse
+				}
+
+				username = answers
+			}
+
+			if !cmd.Flags().Changed("password") {
+				answers, err := utils.AskPassword(msg.AskPassword)
+
+				if err != nil {
+					logger.Debug("Error while parsing answer", zap.Error(err))
+					return utils.ErrorParseResponse
+				}
+
+				password = answers
+			}
+
+			client, err := token.New(&token.Config{Client: f.HttpClient})
+			if err != nil {
+				return err
+			}
+
+			resp, err := client.Create(b64(username, password))
+			if err != nil {
+				return err
+			}
+
+			tokenValid, err := client.Validate(&resp.Token)
+			if err != nil {
+				return err
+			}
+
+			if !tokenValid {
+				return errors.New("token invalid")
```

Comment:
Create a message variable in errors.go file. 
Also: @gabriel-azion can you come up with a message? The token was created BUT it is invalid?

Reply:
@gabriel-azion can you give me an example of the message?

Reply:
`Invalid token detected. The generated token appears to be corrupted or expired. Please check your authentication credentials and generate a new token to proceed.`

Reply:
@PatrickMenoti @maxwelbm does it match the situation?

Reply:
Sounds good to me